Mathlib/CategoryTheory/Limits/FunctorCategory.lean

```diff
@@ -445,3 +446,127 @@ def colimitIsoSwapCompColim [HasColimitsOfShape J C] (G : J ⥤ K ⥤ C) :
 #align category_theory.limits.colimit_iso_swap_comp_colim CategoryTheory.Limits.colimitIsoSwapCompColim
 
 end
+
+-- variable {C : Type u} [Category.{v} C] {D : Type u'} [Category.{v'} D]
+
+--def typefunc (C : Type u) [Category.{v} C] : Type (max v (u + 1)) := C ⥤ (Type u)
+
+--instance blah : Quiver (typefunc C) := sorry
+
+namespace FunctorProd
+
+/-- `prod X Y` is the explicit binary product of simplicial sets `X` and `Y`. -/
+def prod (F G : C ⥤ (Type u)) : C ⥤ (Type u) where
```

Comment:
There is no need for parentheses around `Type u` here. Also, it is important `u` is an arbitrary universe: `Type w` would be better.